### ISSUE TITLE
Question about example html, ChainAbstractionLayer is not defined

### ISSUE THREAD

**Issue by gdhucoder** [Turing-Space/BTC-ETH-Cross-Chain-Atomic-Swap#1]
Hi! 
Thank you for your code about atomic swap!
I followed your install instruction to try btc to eth. But there is a problem occured. I need your help.
![not defined](https://user-images.githubusercontent.com/24959965/69945404-6d997180-1524-11ea-9ee1-6337cc74b75b.png).

**ChainAbstractionLayer is not defined.**

I tried the following steps:
- build bitcoin core. btc works fine.
- install ganache-cli, ganache also works.
- clone your repo
- install chainabstractionlayer
- install a http-server, and the sever works.

And can you make your instruction more specific? I find it is hard to follow.

**Comment by DreamXvan** [comment]
Hello, I encountered the same issue as you. I was able to resolve it by choosing to download the MetaMask extension. However, I’ve now run into a new problem, and I’m not sure whether you experienced this as well or what your specific steps were at the time.

<img width="1101" height="99" alt="Image" src="https://github.com/user-attachments/assets/eb074995-1ad1-467b-8619-4f0af8a0ed04" />
